egs/voxforge/asr1/run_rnnt.sh

```diff
@@ -43,6 +38,11 @@ tag="" # tag for managing experiments.
 
 . utils/parse_options.sh || exit 1;
 
+# datasets
```

Comment:
Thanks!
The [other recipe](https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/run.sh#L69) put the dataset variables after the bash environment setting
```
# Set bash to 'debug' mode, it will exit on :
# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
set -e
set -u
set -o pipefail

train_set=train_960
train_dev=dev
recog_set="test_clean test_other dev_clean dev_other"
```

So, this way is more consistent.
Could you do that?

Reply:
yes, sure